test/spec/Feature/Query/RpcSpec.hs

```diff
@@ -24,50 +24,65 @@ spec :: PgVersion -> SpecWith ((), Application)
 spec actualPgVersion =
   describe "remote procedure call" $ do
     context "a proc that returns a set" $ do
-      it "returns paginated results" $ do
-        request methodPost "/rpc/getitemrange"
-                (rangeHdrs (ByteRangeFromTo 0 0))  [json| { "min": 2, "max": 4 } |]
-           `shouldRespondWith` [json| [{"id":3}] |]
-            { matchStatus = 200
-            , matchHeaders = ["Content-Range" <:> "0-0/*"]
-            }
-        request methodGet "/rpc/getitemrange?min=2&max=4"
-                (rangeHdrs (ByteRangeFromTo 0 0)) ""
-           `shouldRespondWith` [json| [{"id":3}] |]
-            { matchStatus = 200
-            , matchHeaders = ["Content-Range" <:> "0-0/*"]
-            }
-        request methodHead "/rpc/getitemrange?min=2&max=4"
-            (rangeHdrs (ByteRangeFromTo 0 0)) ""
-          `shouldRespondWith`
-            ""
-            { matchStatus = 200
-            , matchHeaders = [ matchContentTypeJson
-                             , "Content-Range" <:> "0-0/*" ]
-            }
+      context "returns paginated results" $ do
+        it "using the Range header" $
+          request methodGet "/rpc/getitemrange?min=2&max=4"
+                  (rangeHdrs (ByteRangeFromTo 1 1)) mempty
+             `shouldRespondWith` [json| [{"id":4}] |]
+              { matchStatus = 200
+              , matchHeaders = ["Content-Range" <:> "1-1/*"]
```

Comment:
Just noticed that our `Content-Range` also doesn't comply the spec https://www.rfc-editor.org/rfc/rfc9110.html#name-content-range. This can be fixed in another PR.

Reply:
Forgot that this was addressed on https://github.com/PostgREST/postgrest/pull/2204. Nevermind.